.circleci/config.yml

```diff
@@ -124,3 +131,23 @@ jobs:
             cd api/python
             pip install twine
             twine upload dist/*
+
+  deploy_aws:
+      docker:
+        - image: circleci/python:3.6-jessie
+      steps:
+        - checkout
+        - run:
+            name: install awscli
+            command: |
+              virtualenv venv
+              . venv/bin/activate
+              pip install --upgrade pip
+              pip install awscli
+        - run:
+            name: deploy cloud formation template
+            command: |
+              . venv/bin/activate
+              chmod +x .circleci/conditional_deploy.sh
+              chmod +x deploy.sh
+              AWS_ACCESS_KEY_ID=$AWS_ACCESS AWS_SECRET_ACCESS_KEY=$AWS_SEC_KEY AWS_DEFAULT_REGION=$AWS_REGION .circleci/conditional_deploy.sh
```

Comment:
which credentials are we using? there should be special credentials only for CI and those credentials will also need review.

Reply:
there's a circleci user in AWS now. I believe that the permissions are probably too generous, but it needs a lot to bring a stack up (modifying endpoints, etc). There's a backlog task to track locking it down in the future for the milestone.
https://app.asana.com/0/979243376089634/1107500789451509/f